index.html

```diff
@@ -41,7 +41,7 @@
 
       (function() {
         var ga = document.createElement('script'); ga.type = 'text/javascript'; ga.async = true;
-        ga.src = ('https:' == document.location.protocol ? 'https://ssl' : 'http://www') + '.google-analytics.com/ga.js';
+        ga.src = 'js/ga.js';
```

Comment:
Yes. Thank you! I forgot about the changes that I had to make to be able to run locally and have no switched all of those back